styles.css

```diff
@@ -62,9 +62,291 @@ body {
+  border-radius: 8px;
+}
+
+.left-block {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0;
+  width: 295px;
+  height: 288px;
+  left: 16px;
+  top: 248px;
+  margin: 12px 0;
+}
+
+.project-title {
+  font-weight: 700;
+  font-size: 24px;
+  color: #344563;
+}
+
+.summary-frame {
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+}
+
+.client {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.role {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.year {
+  margin-right: 10px;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.project-description p {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+}
+
+.project-button {
+  position: relative;
+  display: inline-block;
+  padding: 12px;
+  width: 130px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  align-items: center;
+  line-height: 24px;
+  color: #396df2;
+  margin: 0 10px;
+  cursor: pointer;
+  right: 10px;
+}
+
+.project-button:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.about-myself {
+  background: #fff;
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+}
+
+.username {
+  color: #172b4d;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+}
+
+.get-resume-button {
+  display: inline-block;
+  width: 150px;
+  height: 48px;
+  border: 1px solid #6070ff;
+  box-sizing: border-box;
+  border-radius: 8px;
+  background: #fff;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  align-items: center;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+  margin: 12px 8px;
+}
+
+.get-resume-button:hover {
+  background: #6070ff;
+  color: #fff;
+}
+
+.skill-list {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  margin: 12px 8px;
+  color: #344563;
+  right: 70px;
+}
+
+.skills-list_frame {
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+}
+
+.language-container {
+  display: flex;
+  height: 72px;
+  align-items: center;
+  justify-content: space-between;
+  color: #000;
+  font-style: 500;
+  font-size: 20px;
+  line-height: 20px;
+}
+
+.language-container img {
+  padding: 8px;
+}
+
+.languages {
+  display: inline-block;
+  position: relative;
+}
+
+.language_items {
+  padding: 0;
+}
+
+.language_items li {
+  list-style: none;
+  display: flex;
+  align-items: center;
+  height: 66.72px;
+  background: #f7f7f9;
+  margin: 12px 0;
+  border-radius: 8px;
+}
+
+.tags ul li {
```

Comment:
Please use the good `font-size` to stick to Figma template, actually, your `font-size` is bigger.

<img width="199" alt="image" src="https://user-images.githubusercontent.com/7243132/170822756-7c70270e-b959-4cd3-905e-e2f3f91cd88e.png">

<img width="223" alt="image" src="https://user-images.githubusercontent.com/7243132/170822783-8ebe3bf0-6f62-4b10-b69c-4c5a9a41c062.png">